crates/rome_js_syntax/src/jsx_ext.rs

```diff
@@ -135,6 +137,71 @@ impl JsxSelfClosingElement {
     ) -> SyntaxResult<Option<JsxAttribute>> {
         find_attribute_by_name(self.attributes(), name_to_lookup)
     }
+
+}
+
+
+macro_rules! init_none_array {
+    (@ALWAYSNONE, $name:literal) => {None};
+    (@NAMECOMPARISON, $captures:expr, $i:expr, $att:expr, $element_name:expr, $head:literal,) => {
+        if $element_name == $head {
+            $captures[$i] = Some($att);
+        }
+    };
+    (@NAMECOMPARISON, $captures:expr, $i:expr, $att:expr, $element_name:expr, $head:literal, $($tail:literal,)*) => {
+        if $element_name == $head {
+            $captures[$i] = Some($att.clone());
+        }
+
+        jsx_attributes!(@NAMECOMPARISON, $captures, $i + 1, $att, $element_name, $($tail,)*);
+    };
+    ($n:expr) => {{
+        use rome_rowan::AstNodeList;
+        let mut captures = [$(jsx_attributes!(@ALWAYSNONE, $names),)*];
+        for att in $attributes.iter() {
+            if let Some(attribute) = $crate::JsxAttribute::cast_ref(att.syntax()) {
+                if let Some(name) = attribute.name().ok()
+                    .and_then(|x| $crate::JsxName::cast_ref(x.syntax())) 
+                    .and_then(|x| x.value_token().ok())
+                {
+                    let name = name.text_trimmed();
+                    jsx_attributes!(@NAMECOMPARISON, captures, 0, attribute, name, $($names,)*);
+                }
+            }
+        };
+        captures
+    }};
+}
+
+impl JsxAttributeList {
+    pub fn find_attributes_by_name<const N: usize>( &self, names_to_lookup: [&str; N]) -> [Option<JsxAttribute>; N]
+    {
+        let mut captures: [Option<JsxAttribute>; N] = unsafe {
+            let mut arr: MaybeUninit<_> = MaybeUninit::uninit();
+            for i in 0..N {
+                (arr.as_mut_ptr() as *mut Option<JsxAttribute>).add(i).write(None);
+            }
+            arr.assume_init()
+        };
```

Comment:
This workaround seems to work (don't ask me why)

```suggestion
    {
        const INIT: Option<JsxAttribute> = None;
        let mut captures: [Option<JsxAttribute>; N] = [INIT; N];
```

[source](https://github.com/rust-lang/rust/issues/44796#issuecomment-967747810)

Reply:
Done.